packages/blocks/src/routes/+page.svelte

```diff
@@ -12,8 +13,23 @@ const fetchPointcloud = async () => {
 
 </script>
 
-<div class='relative w-full h-[300px] p-12'>
-  {#await fetchPointcloud() then pointcloud}
-    <SlamMap2D {pointcloud} {motionPath} />
-  {/await}
-</div>
+<div class='m-auto max-w-6xl flex flex-col gap-6 py-6'>
+  <NavigationMap />
+
+  <div class='px-12'>
+    <div class='relative w-full border border-gray-200 aspect-video'>
+      {#await fetchPointcloud() then pointcloud}
+        <SlamMap2D
+          {pointcloud}
+          {motionPath}
+        />
+      {/await}
+    </div>
+  </div>
+
+  <div class='px-12'>
+    <div class="relative w-full pt-0 border border-gray-200 aspect-video">
+      <MapLibre />
+    </div>
+  </div>
+</div>
```

Comment:
```suggestion
</div>

```

Reply:
hmmmm maybe check is not working in CI for the blocks package

Reply:
Yeah I agree, I'm getting some weirdness locally, too